### PR TITLE
[Keyboard] Added support for new keyboard - Ishi80 Mk0

### PR DESCRIPTION
**Pull request by jasonj2232**
## Board/Shield Check-list
 - [x] This board/shield is tested working on real hardware
 - [x] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
 - [x] `.zmk.yml` metadata file added
 - [x] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
 - [x] General consistent formatting of DeviceTree files
 - [x] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
 - [x] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
 - [ ] If split, no name added for the right/peripheral half
 - [x] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
 - [ ] `.conf` file has optional extra features commented out
 - [x] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

**Comment by caksoylar** [comment]
The files look good to me, however is this generally available to buy or shipped as part of a group buy? Otherwise it is encouraged to use a `zmk-config` repo containing the definition that users can fork from.

**Comment by jasonj2232** [comment]
> The files look good to me, however is this generally available to buy or shipped as part of a group buy?

It'll be part of a group buy. Waiting on firmware to be accepted here and elsewhere to go ahead with the GB itself.

**Comment by petejohanson** [comment]
> > The files look good to me, however is this generally available to buy or shipped as part of a group buy?
> 
> It'll be part of a group buy. Waiting on firmware to be accepted here and elsewhere to go ahead with the GB itself.

To be honest, based on the website, it seems like you may be planning a smaller GB, or perhaps a one off. In that scenario, you might consider a separate config repo that users can fork; long term we would like to head that direction with this, and not simply merge every keyboard to the central repo. You can see an example here: https://github.com/petejohanson/revxlp-config

File: app/boards/shields/argoworks_ishi80_mk0/argoworks_ishi80_mk0.keymap [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            &kp KP_ENTER   &kp KP_N3         &kp KP_N2       &kp KP_N1          &kp LSHFT   &kp Z      &kp X           &kp C    &kp V    &kp B    &kp N      &kp M    &kp COMMA  &kp DOT    &kp FSLH               &kp RSHFT
+                           &kp KP_DOT        &kp KP_N0                          &kp LCTRL   &kp LGUI   &lt LOWER LALT                             &kp SPACE                                 &kp RALT   &kp RGUI    &kp K_APP   &kp RCTRL   &kp C_MUTE   
+            >;
+        };
+
+        lower_later{
+            bindings = <
+            &trans        &kp PSCRN      &kp SLCK       &kp PAUSE_BREAK       &kp GRAVE   &kp F1   &kp F2   &kp F3   &kp F4   &kp F5   &kp F6   &kp F7   &kp F8   &kp F9   &kp F10   &kp F11     &kp F12     &kp DEL
+            &reset        &trans         &trans         &trans                &trans      &trans   &trans   &trans   &trans   &trans   &trans   &trans   &trans   &trans   &trans    &trans      &trans      &trans
```
[the remark this answers is not in the document]
Comment (jasonj2232) [reply app/boards/shields/argoworks_ishi80_mk0/argoworks_ishi80_mk0.keymap:41]: Ah this is embarrassing! I didn't realise this PR was still open, I had intended to close it long back. I shall do so now.

**Pull request closed without merging**